src/tools/tidy/src/features.rs

```diff
@@ -164,13 +165,14 @@ pub fn check(path: &Path, bad: &mut bool) {
     });
 
     // FIXME get this whitelist empty.
+    // Especially, only remove things.
```

Comment:
I don't think we need this comment (i.e. the `FIXME` is sufficient). @abonander and I found and modified `tools/tidy/src/features.rs` only after exhausting all other ways to fix the tidy error.